index.bs

```diff
@@ -1012,6 +1032,33 @@ The {{ScopedCredentialEntity}} dictionary describes a user account or a [=relyin
         a user's avatar or a [=relying party=]'s logo.
 </div>
 
+
+### Authenticator Selection Criteria ### {#authenticatorSelection}
+
+The {{AuthenticatorSelectionCriteria}} dictionary describes a set of criterias for which the [RPS] can use to find the appropriate
+authenticator 
+
+<xmp class="idl">
+    dictionary AuthenticatorSelectionCriteria {
+        Attachment    attachment;
+        boolean       requireResidentKey;
```

Comment:
Would requireResidentKey mean that the Authenticator is NOT allowed to offload storage of wrapped key material to (a) the client platform or to (b) remote entities?